app/views/plans/_selector_widget.html.erb

```diff
@@ -1,3 +1,6 @@
+<%# DELETEME !!! %>
+<% raise RuntimeError, 'If this does not throw, then this template is not rendered' %>
```

Comment:
I guess this is for tests, right? so, we can delete the template then? :grimacing:

Reply:
🙉 gosh!